examples/ts/margin-loan-borrow-buy-sell-repay.ts

```diff
@@ -33,7 +33,7 @@ async function example () {
             console.log ('hmm, I have only ', balance_margin[symbol][collateral_coin]['free'], ' in balance, but ', needed_collateral_amount, ' collateral is needed. I should transfer ', needed_collateral_amount, ' from spot');
             // let's check if we have spot balance at all
             const balance_spot = await exchange.fetchBalance ({ 'type': 'spot' });
-            if (balance_spot[collateral_coin]['free'] < needed_collateral_amount) {
+            if (Number (balance_spot[collateral_coin]['free']) < needed_collateral_amount) {
```

Comment:
why are we using Number(...) here? Not sure if this is properly transpiled

Reply:
it was due to a typing error when running tsc.. It looks like it gets ignored in the transpiled files, but just in case I substitued for a `exchange.parseNumber ()`